src/themes/news.json

```diff
@@ -0,0 +1,487 @@
+{
+	"attribution": {
+		"styles": {
+			"default": {
+				"color": "var(--text-color)",
+				"font-size": "var(--body-font-size-small)",
+				"line-height": "var(--body-line-height-small)"
+			}
+		}
+	},
+	"badge": {
+		"styles": {
+			"default": {
+				"align-self": "baseline",
+				"border-radius": "var(--border-radius-pill)",
+				"color": "var(--global-white)",
+				"font-size": "var(--body-font-size-tiny)",
+				"font-weight": "var(--global-font-weight-7)",
+				"line-height": "var(--body-line-height-tiny)",
+				"padding": "var(--global-spacing-1) var(--global-spacing-2)"
+			}
+		}
+	},
+	"badge-danger": {
```

Comment:
Compare to https://github.com/WPMedia/arc-themes-components/pull/177#discussion_r1165505398 and
https://github.com/WPMedia/arc-themes-components/pull/177#discussion_r1165510355